### DISCUSSION TITLE
紐約市教育局禁止學校使用ChatGPT

### DISCUSSION THREAD

**Discussion by Zoe06060606** [1101-GS4719-C/Python-Community-GS4719-C#1085] [General]
文章中提到紐約市教育局下令，禁止學校設備登入或使用ChatGPT，希望能減少學生利用ChatGPT作弊或是毫無思考的得到答案。
ChatGPT的確能給予人類生活很多方便，但的確會延伸相關的問題就像文章提到學生作業只需要跟ChatGPT詢問問題就能照抄完成作業，抑或是作文只要給他關鍵字就可以寫出來，但其實在ChatGPT還未發行前就有給關鍵字產生文章的網站出現--唬爛產生器
https://howtobullshit.me，有段時間還流行於網上，他甚至還能給字數要求，雖然有時候寫出來的文意沒什麼內容就是了🤣
我自己覺得要作業完全限制學生無法使用ChatGPT不太可能，但考試方面就只能盡力避免，例如使用實體考試的方式。
https://news.tvbs.com.tw/tech/2010330

**Comment by peggy0329** [comment]
前幾天才看到一篇在討論有人使用ChatGPT寫論文的文章
我覺得紐約教育局這次的動作其實就已經決定了立場
不知道會不會帶動其他城市的政府在教育方面也做有所限制的科技使用度

**Comment by AnAnistired** [comment]
不得不說chatGPT真的很強，聽說有些電機系的同學都考它完成計概作業，我同學問它微積分它也可以詳細解題，比photomath還屌

**Comment by Twenty0522** [comment]
科技日新月異，要完全防止作弊的學生困難較高，或許可以轉而培養學生對於問題思辨的興趣，以及自己動手解決問題的能力。

**Comment by Hahahahbla** [comment]
你提到的問題非常值得討論。紐約市教育局禁止在學校設備上使用ChatGPT，確實是出於擔心學生會利用它來作弊或避免自主思考。這一決定反映了AI技術在教育領域應用中的一個重要挑戰。

確實，ChatGPT和類似工具可以極大地方便人們的生活和學習，但也可能引發一些倫理和學術誠信問題。

總的來說，AI技術帶來了很多便利，但我們也需要謹慎應對其帶來的挑戰。希望通過合理的政策和教育，能夠更好地利用這些技術，同時維護學術誠信和教育質量。

**Comment by OG-Matcha** [comment]
我覺得與其去限制學生使用 AI 工具，不如教學生如何「正確」的使用這些應用。工具所產生的內容並非完全正確，如果一味的依賴他們而沒有去嘗試理解或思考其中的內容是無法成長的，甚至會比原本更糟糕，這就不是在使用 AI 加快效率，根本就是被 AI 戲弄。

**Comment by happy246810** [comment]
我覺得說與其完全禁止使用AI工具，應該是要全面開放使用，畢竟假如現在不使用，那在出社會後，這些學生就少了一項原本就要擁有的能力。而考試的部分，則是請老師盡量出一些向申論題這種需要學生用自己的話去解釋的題目。

**Comment by NcuJerryKao** [comment]
針對您提供的文章討論，以下是一些建設性的建議：
1. 技術整合與教育目標：教育部門可以考慮如何將ChatGPT與其他AI技術整合到教學策略中，而不僅僅是禁止使用。例如，設計特定的課程或活動，利用ChatGPT來促進學生的批判性思維和創造力，如設定特定問題讓學生透過AI來探索多種答案，並討論其背後的邏輯和原因。
2. 教育AI的正當用途：教育機構可以開設關於人工智能的課程，教育學生如何負責任地使用這些技術。包括教授學生辨識資訊真偽的技能，以及如何有效地利用AI作為學習輔助工具。
3. 創新評量方法：重新考慮評量學生學習成果的方法。隨著技術的進步，傳統的考試和作業可能不再足夠或適用。學校可以開發更多基於實際表現和項目的評估方式，如項目合作、口頭報告和實踐操作等，這些方式較難通過AI工具來簡單代替。
4. 強調學習過程：教育者應該強調學習過程的重要性，而不僅僅是結果。透過討論、反思和批評性思考的教學方法來鼓勵學生深入理解學習材料，而不是僅僅尋求答案。
5. 家長和社區參與：教育機構應該與家長和社區合作，共同開發策略來應對AI技術對學習的影響。這可以包括家長教育課程，幫助他們理解孩子可能使用的技術工具，以及如何在家中監督和指導孩子使用這些技術。
通過這些建議，學校不僅能夠解決學生利用AI作弊的問題，還能夠利用這些先進技術來豐富教學內容和方法，最終提升教育質量和學生的學習體驗。

**Comment by jylin34** [comment]
我認為這種規定很難實行，也不應該實行，教育局要做的應該是改變教學方式，讓老師教育學生如何使用生成式語言模型幫助學習、生活，並教導如何將其提供的資訊內化成自己的知識才對，讓學生認知到這是一個幫助我們快速成長的工具。

**Comment by titan92601** [comment]
我認為紐約市教育局禁止學校設備使用ChatGPT，以防學生作弊，但完全限制不切實際。應引導學生正確使用AI，培養批判性思維和創造力。通過實體考試和創意作業設計減少作弊，加強學術道德教育。這樣能平衡技術優勢和學術誠信，促進學生全面發展。

**Comment by sandyho0307** [comment]
我覺得在這個技術快速發展的時代，教育者需要找到一種平衡，利用技術的力量來增強教育，同時確保學生能夠獨立思考和學習。

**Comment by buoioiry** [comment]
在使用chat gpt的這些經驗中，其實我覺得比起給予正確解答，chatgpt更像是給一些搜尋的方向，因為有些科目使用他還是會有種唬爛唬爛的感覺，但可以讓學生更清楚可以從哪裡切入問題去解決，或許更應該做的是引導學生使用ai技術，而非完全限制。

**Comment by tshsu04224** [comment]
ChatGPT真的很厲害，現在有關程式的作業真的沒有他的幫忙可能寫不太出來，有點沒辦法想像要是作業無法使用ChatGPT幫忙，期末的我被各種專題轟炸會比現在痛苦多少倍。

**Comment by cassy0201** [comment]
我覺得Chat GPT是一個很好的學習輔助工具，讓學習更高效率且多元，直接下令學生禁止使用有點治標不治本，感覺可以從教育策略上讓學生學習如何利用Chat GPT來進行更深入的知識探索

**Comment by OHoln** [comment]
雖然很多學校或教授認為chatgpt會讓學生不會思考，但我個人認為chatgpt可以很好的輔佐學生學習，省去一大部分在網路瀏覽翻找資料的時間，其實是一個很有效率的學習方法，缺點就是面對一些專業問題他的答案可能會有偏差就是了。

**Comment by 710388** [comment]
應該是認為太早接觸會影響到學生思辨能力吧

**Comment by yubo0oo** [comment]
我覺得禁用絕對不是一個好方法
應該要學回如何善用
並最大化利用，這才是正確的共存方式
